src/pages/Bestellung.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+export default function Bestellung({ localStorage, inputData, totalPrice }) {
+  console.log(localStorage);
+  return (
+    <>
+      <Section>
+        <h4>Prüfe deine daten und bestätige den Kauf</h4>
+        <main>
+          <h4>Dein persönliches Data: </h4>
+          <h5>Name: {inputData.name}</h5>
+          <h5>Email: {inputData.email}</h5>
+          <h5>Adresse: {inputData.address}</h5>
+          <h4>
+            Deie bestellte Artikel:
+            {localStorage.map((element) => (
```

Comment:
This will confuse a lot of people, seeing the `map` method with `localStorage`

Reply:
already done, thanks